sync2jira/downstream_pr.py

```diff
@@ -44,10 +44,10 @@ def format_comment(pr, pr_suffix, client):
     try:
         ret = client.search_users(pr.reporter)
     except Exception as ex:
-        print("search users failed")
-        ret=[]
+        log.info("Failed to fetch user for the reporter %s with exception %r", pr.reporter, ex)
```

Comment:
You probably don't want to apply a `%r` to an exception object.  You probably want to coerce the object to a string, e.g., `str(ex)`.